### ISSUE TITLE
java.io.IOException: error response: {faultcode=s:Client, faultstring=UPnPError, UPnPError/errorCode=720, UPnPError/errorDescription=InvalidDeviceUUID}

### ISSUE THREAD

**Issue by dotStart** [LordAkkarin/Beacon#21]
The application crashes due to an error response from the router.

**Reproduction Steps**
??? - Illegal router reply

**Expected Behavior**
An error message should appear.

**Environment (please complete the following information):**
 - OS: Windows 10
 - Application Version: 2.4.0

**Additional Context**
```
Message: CompletionException: java.io.IOException: error response: {faultcode=s:Client, faultstring=UPnPError, UPnPError/errorCode=720, UPnPError/errorDescription=InvalidDeviceUUID}
  java.util.concurrent.CompletableFuture.reportJoin::-1
  java.util.concurrent.CompletableFuture.join::-1
   
  tv.dotstart.beacon.forwarding.ActionExtensionsKt.invoke in ActionExtensions.kt::48
  tv.dotstart.beacon.forwarding.PortMapping.refresh in PortMapping.kt::67
  tv.dotstart.beacon.forwarding.InternetGatewayDevice.forwardPort in InternetGatewayDevice.kt::81
  tv.dotstart.beacon.forwarding.PortExposureProvider.expose in PortExposureProvider.kt::86
  tv.dotstart.beacon.controller.MainController.onServiceOpen in MainController.kt::198
  jdk.internal.reflect.NativeMethodAccessorImpl.invoke0::-2
  jdk.internal.reflect.NativeMethodAccessorImpl.invoke::-1
  jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke::-1
  java.lang.reflect.Method.invoke::-1
  com.sun.javafx.reflect.Trampoline.invoke::-1
   
  jdk.internal.reflect.GeneratedMethodAccessor2.invoke::-1
  jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke::-1
  java.lang.reflect.Method.invoke::-1
  com.sun.javafx.reflect.MethodUtil.invoke::-1
  com.sun.javafx.fxml.MethodHelper.invoke::-1
  javafx.fxml.FXMLLoader$MethodHandler.invoke::-1
  javafx.fxml.FXMLLoader$ControllerMethodEventHandler.handle::-1
  com.sun.javafx.event.CompositeEventHandler.dispatchBubblingEvent::-1
  com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent::-1
  com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent::-1
  com.sun.javafx.event.CompositeEventDispatcher.dispatchBubblingEvent::-1
  com.sun.javafx.event.BasicEventDispatcher.dispatchEvent::-1
  com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent::-1
  com.sun.javafx.event.BasicEventDispatcher.dispatchEvent::-1
  com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent::-1
  com.sun.javafx.event.BasicEventDispatcher.dispatchEvent::-1
  com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent::-1
  com.sun.javafx.event.BasicEventDispatcher.dispatchEvent::-1
  com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent::-1
  com.sun.javafx.event.BasicEventDispatcher.dispatchEvent::-1
  com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent::-1
  com.sun.javafx.event.EventUtil.fireEventImpl::-1
  com.sun.javafx.event.EventUtil.fireEvent::-1
  javafx.event.Event.fireEvent::-1
  javafx.scene.Node.fireEvent::-1
  javafx.scene.control.Button.fire::-1
  com.sun.javafx.scene.control.behavior.ButtonBehavior.mouseReleased::-1
  com.sun.javafx.scene.control.inputmap.InputMap.handle::-1
  com.sun.javafx.event.CompositeEventHandler$NormalEventHandlerRecord.handleBubblingEvent::-1
  com.sun.javafx.event.CompositeEventHandler.dispatchBubblingEvent::-1
  com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent::-1
  com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent::-1
  com.sun.javafx.event.CompositeEventDispatcher.dispatchBubblingEvent::-1
  com.sun.javafx.event.BasicEventDispatcher.dispatchEvent::-1
  com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent::-1
  com.sun.javafx.event.BasicEventDispatcher.dispatchEvent::-1
  com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent::-1
  com.sun.javafx.event.BasicEventDispatcher.dispatchEvent::-1
  com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent::-1
  com.sun.javafx.event.BasicEventDispatcher.dispatchEvent::-1
  com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent::-1
  com.sun.javafx.event.BasicEventDispatcher.dispatchEvent::-1
  com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent::-1
  com.sun.javafx.event.EventUtil.fireEventImpl::-1
  com.sun.javafx.event.EventUtil.fireEvent::-1
  javafx.event.Event.fireEvent::-1
  javafx.scene.Scene$MouseHandler.process::-1
  javafx.scene.Scene$MouseHandler.access$1300::-1
  javafx.scene.Scene.processMouseEvent::-1
  javafx.scene.Scene$ScenePeerListener.mouseEvent::-1
  com.sun.javafx.tk.quantum.GlassViewEventHandler$MouseEventNotification.run::-1
  com.sun.javafx.tk.quantum.GlassViewEventHandler$MouseEventNotification.run::-1
  java.security.AccessController.doPrivileged::-2
  com.sun.javafx.tk.quantum.GlassViewEventHandler.lambda$handleMouseEvent$2::-1
  com.sun.javafx.tk.quantum.QuantumToolkit.runWithoutRenderLock::-1
  com.sun.javafx.tk.quantum.GlassViewEventHandler.handleMouseEvent::-1
   
  com.sun.glass.ui.View.handleMouseEvent::-1
  com.sun.glass.ui.View.notifyMouse::-1
  com.sun.glass.ui.win.WinApplication._runLoop::-2
  com.sun.glass.ui.win.WinApplication.lambda$runLoop$3::-1
  java.lang.Thread.run::-1
Message: IOException: error response: {faultcode=s:Client, faultstring=UPnPError, UPnPError/errorCode=720, UPnPError/errorDescription=InvalidDeviceUUID}
  net.mm2d.upnp.internal.impl.ActionInvokeDelegate.invoke in ActionInvokeDelegate.kt::95
  net.mm2d.upnp.internal.impl.ActionInvokeDelegate.invoke in ActionInvokeDelegate.kt::49
  net.mm2d.upnp.internal.impl.ActionImpl.invokeCustomSync in ActionImpl.kt::49
  net.mm2d.upnp.internal.impl.ActionImpl$invokeInner$1.invoke in ActionImpl.kt::63
  net.mm2d.upnp.internal.impl.ActionImpl$invokeInner$1.invoke in ActionImpl.kt::22
  net.mm2d.upnp.internal.thread.ExecuteFunctionKt$toRunnable$1.run in ExecuteFunction.kt::19
  java.util.concurrent.ThreadPoolExecutor.runWorker::-1
  java.util.concurrent.ThreadPoolExecutor$Worker.run::-1
  java.lang.Thread.run::-1
```

**Comment by dotStart** [comment]
Raygun report https://app.raygun.com/crashreporting/27ce4ol/errors/4811931783 has more details about the error

**Comment by dotStart** [comment]
This issue has already been addressed in v3.0
